docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+services:
+  postgres:
+    container_name: lenra-postgres
```

Comment:
You should use different container names for `lenra-postgres` and `lenra-mongo` because they are conflicting with the Lenra server's container names. Maybe we could set them to `devtool-postgres` and `devtool-mongo`.

```console
jonas@lenra:~/Documents/devtool$ docker compose up -d
[+] Running 1/0
 ✔ Network devtool_default   Created                                                                                                                             0.1s 
 ⠋ Container lenra-mongo     Creating                                                                                                                            0.0s 
 ⠋ Container lenra-postgres  Creating                                                                                                                            0.0s 
Error response from daemon: Conflict. The container name "/lenra-mongo" is already in use by container "f49368bba130a70c0321242ba8f97a8075d4e97a6546ed4eeba7f4680a6fe948". You have to remove (or rename) that container to be able to reuse that name.
```

Reply:
I don't think that it's a real problem since if you have the server ones, you don't need this one and you will have conflict on exposed ports.